internal/config/constants.go

```diff
@@ -54,9 +54,10 @@ const (
 	EnvSiteName   = "MINIO_SITE_NAME"
 	EnvSiteRegion = "MINIO_SITE_REGION"
 
-	EnvMinIOSubnetLicense = "MINIO_SUBNET_LICENSE" // Deprecated Dec 2021
-	EnvMinIOSubnetAPIKey  = "MINIO_SUBNET_API_KEY"
-	EnvMinIOSubnetProxy   = "MINIO_SUBNET_PROXY"
+	EnvMinIOSubnetLicense    = "MINIO_SUBNET_LICENSE" // Deprecated Dec 2021
+	EnvMinIOSubnetAPIKey     = "MINIO_SUBNET_API_KEY"
+	EnvMinIOSubnetProxy      = "MINIO_SUBNET_PROXY"
+	EnvMinIOConsoleSubnetURL = "MINIO_CONSOLE_SUBNET_URL"
```

Comment:
if you really want to add this environment variable must be part of the `subnet` subsystem, since this is testing my suggestion is it is better to rely on `console` standalone for testing and use the necessary environment variable.

Reply:
As earlier discussed ,  we are not promoting standalone console. 
On one side we set `CONSOLE_SUBNET_LICENSE` in MinIO so that the embedded console reads it and parses the license. Since the embedded console uses SUBNET prod url to get the public key, The license parsing fails against local subnet server . 

Now if i introduce a  envt variable in console and try against a standalone console, then i don't get the value of `CONSOLE_SUBNET_LICENSE`  in standalone console server as it was set in MinIO server as envt variable to be used  by embedded console not the standalone one.

So to solve this i introduced this `MINIO_CONSOLE_SUBNET_URL`.

For MinIO users it doesnt make any sense , but yes for testing  CONSOLE with MINIO , its needed.

Reply:
Its quiet similar to `mc license info alias --dev` . Dev is used against the local SUBNET server .

Reply:
Yeah but we can't add such random environment variables for testing alone they have to really serve a good purpose, it has to follow a set guideline if this is simply an internal value it must be `_MINIO` that is it is meant for testing alone. 

@sinhaashish "standalone" console is always used in testing things quickly without having to rebuild MinIO. So this is still a valid case.

What is not promoted running "standalone" console is for customers, we are free to use it the way we please for testing and working on functionality.

Reply:
We can make use of the existing env variable `MINIO_CI_CD`. When it is set, the SUBNET URL can be hard coded to `http://localhost:9000`

Reply:
closing this PR, as we can test the console SUBNET URL by changing the value in code. In alignment with Harsha, As far as `MINIO_CI_CD` is concerned we can not use it as it is used for figure out root disk . 

Run server `./minio server /tmp/{2...5}` without setting `MINIO_CI_CD`  you will get the error  `Drive `/tmp/2` is part of root drive, will not be used (*errors.errorString)`